site/content/en/docs/commands/pause.md

```diff
@@ -7,11 +7,12 @@ description: >
 
 ## minikube pause
 
-pause Kubernetes
+Pauses a running local Kubernetes cluster
 
 ### Synopsis
 
-pause Kubernetes
+Pauses local Kubernetes cluster and free up CPU resources 
```

Comment:
Let’s add this, to the description
“Without pausing pods/apps inside the cluster.”

Reply:
As in "Pauses local Kubernetes cluster and free up CPU resources, without pausing pods/apps inside the cluster" ?

Reply:
I believe "minikube pause", pauses all the applications running in k8s cluster and the k8s cluster itself. Please correct me if I am wrong.